src/test/resources/features/addEducation.feature

```diff
@@ -0,0 +1,11 @@
+@ui @regression
+Feature: Add Education
+
+  Scenario: Title of your scenario
```

Comment:
Tag the scenario, not the feature.
And the title of scenario doesn't make sense.